app/_locales/en/messages.json

```diff
@@ -2971,6 +2971,21 @@
   "notifications24Title": {
     "message": "Advanced gas fees by network"
   },
+  "notifications29ActionText": {
+    "message": "Got it"
```

Comment:
we have a few of instances of this message already. can we re-use the 'gotIt' key for this instead of creating another one?

Reply:
It gets confusing to use other keys like `notifications20ActionText`

Reply:
Done

Reply:
not sure if there is a PR open for this yet, but we'll want to update the `gotIt` translation key for all language files to something like 'gotItExclamation' and condense the instances mentioned to "gotIt"